contracts/modules/capital/Pool.sol

```diff
@@ -156,6 +166,30 @@ contract Pool is IPool, MasterAwareV2, ReentrancyGuard {
     return swapDetails[assetAddress];
   }
 
+  function getAssetId(address assetAddress) public view returns (uint) {
+    uint assetCount = assets.length;
+    for (uint i = 0; i < assetCount; i++) {
+      if (assets[i].assetAddress == assetAddress) {
+        return i;
+      }
+    }
+    revert("Pool: Asset not found");
```

Comment:
Is it worth taking this chance to convert the reverts to custom errors? There are 8 reverts in this contract